modules/exploits/multi/php/jorani_path_trav.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Jorani unauthenticated Remote Code Execution',
+                      'Description' => '
+          This module exploits an unauthenticated Remote Code Execution in Jorani.
+          It abuses 3 vulnerability: log poisonning and redirection bypass via header spoofing, then it uses path traversal to trigger the vulnerability.
+          It has been tested on Jorani 1.0.0.
+        ',
+                      'License' => MSF_LICENSE,
+                      'Author' =>
+                        [
+                          'RIOUX Guilhem (jrjgjk)'
+                        ],
+                      'References' =>
+                        [
+                          %w[CVE CVE-2023-26469],
```

Comment:
```suggestion
                          %w[CVE 2023-26469],
```

Reply:
Suggested change was applied in https://github.com/rapid7/metasploit-framework/pull/18123/commits/8cae031d9765e7a400adff5fb12efb2c9b5fd441, confirming resolved.